src/main/java/org/jboss/modules/ModuleLoggerFinder.java

```diff
@@ -142,6 +144,24 @@ static void activate(final ClassLoader cl) {
         }
     }
 
+    private static int getMaxLogMessages() {
+        final int dft = 10_000;
```

Comment:
Please note this is a completely arbitrary number and if there is a better default, please let me know.